From: SeongJae Park <sj@kernel.org>
Date: Mon, 9 Jun 2025 12:24:10 -0700
Subject: [PATCH] ==== use alloc_migration_target() ====

mm/damon: use alloc_migration_target() for DAMOS_MIGRATE_{HOT,COLD}

DAMOS_MIGRATE_{HOT,COLD} implementation resembles that for demotion, and
hence the behavior is also similar to that.  But, since those are not
only for demotion but general migrations, it would be better to match
with that for move_pages() system call.  Make the implementation and the
behavior more similar to move_pages() by not setting migration fallback
nodes, and using alloc_migration_target() instead of
alloc_migrate_folio().

alloc_migrate_folio() was renamed from alloc_demote_folio() and been
non-static function, to let DAMOS_MIGRATE_{HOT,COLD} call it.  As
alloc_migration_target() is called instead, the renaming and non-static
changes are no more required but could only make future code readers
be confused.  Revert the changes, too.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_xqxf7ah0 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_xqxf7ah0

diff --git a/hkml_cv_bogus/hkml_cv_bogus_xqxf7ah0 b/hkml_cv_bogus/hkml_cv_bogus_xqxf7ah0
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

